backend/tests/src/tests/json-schemas/auth/sign-in/sign-in-response.schema.ts

```diff
@@ -0,0 +1,23 @@
+const signInResponseSchema = {
+  'type': 'object',
+  'properties': {
+    'token': {
+      'type': 'string',
+    },
+    'user': {
+      'type': 'object',
+      'properties': {
+        'id': {
+          'type': 'integer',
+        },
+        'email': {
+          'type': 'string',
+        },
+      },
+      'required': ['id', 'email'],
+    },
+    'required': ['token', 'user'],
+  },
+};
```

Comment:
1. Auth returns more than just id and email for user. Please see the types and update json schema.
2. This `user` object is a separate interface in shared folder, so please move it to `backend/tests/src/tests/json-schemas/user/user-by-id/user-by-id-response.schema.ts`
3. Same about quotes